### DISCUSSION TITLE
How do I mark a flag as optional?

### DISCUSSION THREAD

**Discussion by krystalcode** [chshersh/iris#128] [Q&A] [answered]
I have successfully used Iris to provide flags for my command. However, whenever I skip any of the flags, an error is raised. Would this be resolved by providing a default value? Is there an example of a recommended way for doing so?

**Comment by chshersh** [comment]
Hi @krystalcode,

Appreciate you showing interest in this project! However, I'm not maintaining it at the moment, so I'm afraid you, it'd be wise not to expect answers to your questions.

Have a great day!

**Reply by krystalcode** [reply, marked as answer]
@chshersh Thank you for the response, good to know that the project is not actively maintained.

To answer my own question and for anybody new looking for this, I was able to solve this by providing a default value as supported by `optparse-applicative` - see Quick Start at https://hackage.haskell.org/package/optparse-applicative-0.16.1.0.
